tests/unit/argument-test.js

```diff
@@ -26,7 +26,7 @@ module('Unit | @argument', function() {
 
     assert.throws(function() {
       Foo.create({ prop: 'wrong type' });
-    }, /Foo#prop expected value of type number during 'init', but received: 'wrong type'/);
+    }, /Foo#prop expected value of type number during 'set', but received: 'wrong type'/);
```

Comment:
A few of these changed because the validation is run as part of the setter before we even get to the `init` stage. I don't think it's a big deal that this has changed, since the only difference from the user's perspective is the error text.

Reply:
Resolving, in case this is just noise during a review.